injective/assetlist.json

```diff
@@ -143,6 +144,7 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/ninja.png"
       },
+      "coingecko_id": "dog-wif-nuchucks",
```

Comment:
I wonder if they intend on correcting this incorrect spelling of nunchucks (missing the second 'n')...

Reply:
i have no idea. want me to remove it for now?